tests/v2/Service/GdprServiceTest.php

```diff
@@ -45,8 +45,8 @@ public function testGetDaysBeforeExpiration(\DateTimeImmutable $updatedAt, int $
 
     public function provideTestGetDaysBeforeExpiration(): \Generator
     {
-        yield 'Should return 364 when password has been updated today' => [new \DateTimeImmutable('now'), 364];
-        yield 'Should return 4 when password has been updated 360 days ago' => [(new \DateTimeImmutable('now'))->sub(new \DateInterval('P360D')), 4];
+        yield 'Should return 364 when password has been updated today' => [new \DateTimeImmutable('now'), 365];
+        yield 'Should return 4 when password has been updated 360 days ago' => [(new \DateTimeImmutable('now'))->sub(new \DateInterval('P360D')), 5];
```

Comment:
You also need to change the value in the sentence  "Should return **364** when password has been updated today"